### ISSUE TITLE
Linux: Instructions to install libGL is Webots is installed from the tarball package

### ISSUE THREAD

**Issue by stefaniapedrazzi** [omichel/webots-doc#458]
Document the installation of of libGL if Webots is installed from the tarball.

Otherwise the phsics plugins cannot be recompiled correctly:
```
make
# updating flying_mybot.d
# compiling flying_mybot.c
In file included from flying_mybot.c:7:0:
/home/stefi/Downloads/webots/include/plugins/physics.h:9:19: fatal error: GL/gl.h: No such file or directory
compilation terminated.
/home/stefi/Downloads/webots/resources/Makefile.include:648: recipe for target 'build/release/flying_mybot.o' failed
make: *** [build/release/flying_mybot.o] Error 1
```

**Comment by stefaniapedrazzi** [comment]
This is also required to recompile robot windows, like for example the e-puck Qt one.

**Comment by stefaniapedrazzi** [comment]
Fixed in #459.
